### PR TITLE
Add jquery-flexdatalist@1.9.5 w/ git auto-update

### PR DESCRIPTION
**Pull request by sophie-li**
Pull request for issue: #10286 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/sergiodlopes/jquery-flexdatalist
 * Official website (optional, not the repository): http://projects.sergiodinislopes.pt/flexdatalist/
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 10
   - Count of stars: 164
   - Count of forks: 24
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery-flexdatalist/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "datalist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sergiodlopes/jquery-flexdatalist.git"
+  },
+  "author": "Sérgio Dinis Lopes",
+  "license": "MIT",
+  "homepage": "https://projects.sergiodinislopes.pt/flexdatalist/",
+  "npmName": "jquery-flexdatalist",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-flexdatalist/package.json]: @sophie-li https://www.npmjs.com/package/jquery-flexdatalist not found. Please transfer to use git auto-update config. You can take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md). Thanks.

File: ajax/libs/jquery-flexdatalist/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "jquery.flexdatalist.js*"
```
Comment (pvnr0082t) [inline ajax/libs/jquery-flexdatalist/package.json]: Let's use "`jquery.flexdatalist*"` here.

File: ajax/libs/jquery-flexdatalist/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "plugin",
+    "datalist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sergiodlopes/jquery-flexdatalist.git"
+  },
+  "author": "Sérgio Dinis Lopes",
+  "license": "MIT",
+  "homepage": "https://projects.sergiodinislopes.pt/flexdatalist/",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-flexdatalist/package.json]: It seems that it doesn't support SSL. Please just use `http`.

Comment (extend1994) [inline ajax/libs/jquery-flexdatalist/package.json]: @pvnr0082t I can browse this homepage via `https`. Maybe you can try again.

Comment (pvnr0082t) [inline ajax/libs/jquery-flexdatalist/package.json]: Okay, I can access now. :+1:

**Comment by sophie-li** [comment]
@maruilian11 will note that, thank you! i am not sure what else it is missing for me to do in this PR so let me know if i miss anything. :)

**Comment by PeterDaveHello** [comment]
@pvnr0082t don't forget to comeback once PR was updated.

**Comment by PeterDaveHello** [comment]
@cdnjs/library-reviewer request review from me once you've got two approvals and the problems were fixed.

**Comment by PeterDaveHello** [comment]
@sophie-li congratulations! 👍

**Pull request merged**